numpy/core/__init__.py

```diff
@@ -3,9 +3,25 @@
 from .info import __doc__
 from numpy.version import version as __version__
 
+import os
+
+# on Windows NumPy loads an important OpenBLAS-related DLL
+# and the code below aims to alleviate issues with DLL
+# path resolution portability with an absolute path DLL load
+if os.name == 'nt':
+    from ctypes import WinDLL
+    from pathlib import Path
```

Comment:
I don't think `pathlib` is in Python 2.7 by default, so this breaks backport.

Reply:
Looks like it was backported with Python 2.7.6, which may be a problem with some folks, but we may need to live with that.

Reply:
True. I can't tell if we're actually going to use this PR or not, so trying to decide if I should make it 2.x compatible or not.

Reply:
Ok, I'll work on this now to see if I can give us the option of a healthier backport.

Reply:
Ok, I've replaced `pathlib` usage with old school `os.path` and friends & confirmed Azure is happy and still prints a warning with multiple DLLs in `.libs`

I'll mark this as resolved for now, but the real test will be on the backport PR if it happens since that should have 2.x Windows still on Azure CI I think.